### DISCUSSION TITLE
Top bar is no longer the accent color?

### DISCUSSION THREAD

**Discussion by Pinsplash** [black7375/Firefox-UI-Fix#1031] [General]
I just updated and my top bar is no longer the OS accent color when the window is in focus. Is this an installation issue or intentional? Cause that was about half the reason I used this.

**Comment by black7375** [comment]
Do you mean photon style's top bar color?

**Comment by TheRavirn** [comment]
Yeah since the latest Firefox release the tab background color is gray instead of OS color. Not the first time this has happened and you've fixed it in a new release.
